.github/instructions/comparison.instructions.md

```diff
@@ -0,0 +1,76 @@
+# Swagger to TypeSpec Migration Instructions - Validation Steps
+
+Follow these steps to guide users through comparing Swagger and TypeSpec specifications after initial conversion to validate the migration result:
+
+## Understanding User Input
+When a user asks for help with comparing Swagger and TypeSpec specifications, they will typically provide the name of the service folder under `specification/` (e.g., `compute`, `network`, `storage`). Always ask for this information if it's not provided.
+
+1. First, determine the absolute path to the generated Swagger files. Examine the `tspconfig.yaml` file in the TypeSpec folder:
+   - Look for the `emitter` configurations, particularly the `@azure-tools/typespec-autorest` emitter
+   - Find the `output-file` option which specifies where Swagger files are generated
+   - The value of {version} is determined from the `main.tsp` file:
+     - Look for an enum named `Versions` or `ApiVersions` in the main.tsp file
+     - The enum values represent the API versions supported by the service
+     - For example: `enum Versions { v2023_01_01 = "2023-01-01" }` 
+     - The string value (e.g., "2023-01-01") is what will be substituted for {version} in the output path
+   - The value of {version-status} is either stable or preview
+   - The value of {service-name} is determined from the `main.tsp` file:
+     - The namespace of the main TypeSpec file.
+     - For example: `namespace Microsoft.Compute;`
+     - The service name is `Microsoft.Compute` in this case.
+
+2. Second, determine the location of the original Swagger folder:
+   - The original Swagger is typically located at `sparse-spec\specification\{service-folder-name}`
+   - Use the innermost folder (e.g., `sparse-spec\specification\storageactions\resource-manager\Microsoft.StorageActions\stable\2023-01-01`) as the original Swagger folder path
+   - Check if this folder exists by running:
+     ```powershell
+     Test-Path sparse-spec\specification\{service-folder-name}
+     ```
+   - If the folder doesn't exist, instruct the user to run:
+     ```powershell
```

Comment:
the script execution failed  
![image](https://github.com/user-attachments/assets/b8aa6f8f-6675-465a-a98b-247549984213)

Reply:
how to root cause the failure?